CHANGELOG.md

```diff
@@ -3,6 +3,116 @@
 
+## 5.0.0-beta.0
+
+_Sep 16, 2021_
+
+🎉 This is the first release of the data grid component with full support of the new MUI 5.0 release 🎉!
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚡ Migrate to the new `@mui/material` and `@mui/styles` packages and drop support for `@material-ui/core` (#2515, #2571) @m4theushw
+- 📚 Migrate to the new documentation infrastructure and design (#2441) (@DanailH, @m4theushw)
+- 🚀 Introduce the row menu feature  (#2385) @m4theushw
+- 🚀 Introduce the `disableVirtualization` prop (#2326) @m4theushw
+- 🚀 Introduce the new `isRowSelected` api method (#2523) @flaviendelangle
+- 🌎 Add Simplified Chinese (zhCN) localization (#2431) @wlf100220
+- 🌎 Add Korean (koKR) localization (#2446) @zzossig
+
+### `@mui/x-data-grid@v5.0.0-beta.0` / `@mui/x-data-grid-pro@v5.0.0-beta.0`
+
+#### Breaking changes
+
+- [DataGridPro] Remove `apiRef.current.getState` method
+
+  ```diff
+  -const state = apiRef.current.getState();
+  +const state = apiRef.current.state
+  ```
+
+- [DataGridPro] Unification of the selection API parameters between `selectRow` and `selectRows`
+
+
+  ```diff
+  -selectRow: (id: GridRowId, isSelected?: boolean, allowMultiple?: boolean = false) => void;
+  +selectRow: (id: GridRowId, isSelected?: boolean, resetSelection?: boolean = false) => void;
+  ```
+
+- [DataGrid] Replace the `options` with direct prop reads
+
+  ```diff
+  -const { options } = useGridSlotComponentProps();
+  +const rootProps = useGridRootProps();
+  ``` 
+
+#### Changes
+
+- [DataGridPro] Only apply `checkboxSelectionVisibleOnly` when pagination is enabled (#2443) @flaviendelangle
+- [DataGridPro] Remove `apiRef.current.getState` method (#2579) @flaviendelangle
+- [DataGrid] Add `disableVirtualization` prop (#2326) @m4theushw
+- [DataGrid] Add missing exports from param models (#2448) @flaviendelangle
+- [DataGrid] Add missing keys to the `classes` prop (#2458) @m4theushw
+- [DataGrid] Add row menu (#2385) @m4theushw
+- [DataGrid] Add zhCN localization (#2431) @wlf100220
+- [DataGrid] Add koKR localization (#2446) @zzossig
+- [DataGrid] Clean the selection public api (#2523) @flaviendelangle
+- [DataGrid] Do not call `useGridColumnResize` and `useGridInfiniteLoader` (#2580) @flaviendelangle
+- [DataGrid] Do not show right border of last column header when its cells don't have it (#2444) @flaviendelangle
+- [DataGrid] Don't consider unselectable rows when `selectionModel` is used (#2464) @m4theushw
+- [DataGrid] Drop v4 support (#2515) @m4theushw
+- [DataGrid] Fire `columnOrderChange` event after state update (#2451) @flaviendelangle
+- [DataGrid] Fix TypeScript type error for toolbar components (#2393) @ZeeshanTamboli
+- [DataGrid] Fix navigation between column headers with rows filtered (#2440) @m4theushw
+- [DataGrid] Force `scrollEndThreshold` to undefined (#2574) @flaviendelangle
+- [DataGrid] Improve jaJP localization (#2502) @daikiojm
```

Comment:
```suggestion
- [DataGrid] Improve Japanese (jaJP) locale (#2502) @daikiojm
```

Reply:
Will you apply the suggestion? We have been using the "French (frFR) locale" format in the past releases.